pvlib/ivtools/sdm.py

```diff
@@ -166,6 +166,9 @@ def fit_desoto(v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc, cells_in_series,
         Reference temperature condition [C]
     irrad_ref: float, default 1000
         Reference irradiance condition [W/m2]
+    init_guess: dict, optional
+        Initial values for optimization, must have keys `'Rsh_0'`, `'a_0'`,
+        `'IL_0'`, `'Io_0'`, `'Rs_0'`.
```

Comment:
Could it be useful to accept a subset of these as input?  (And use the internal values for the rest?)

Reply:
Good suggestion. Convergence of this method appears to be somewhat sensitive to the initial guess and the built-in may not always work.